examples/abitest/module_0/rust/src/lib.rs

```diff
@@ -178,13 +178,14 @@ pub extern "C" fn frontend_oak_main(_in_handle: u64) {
     let _ = std::panic::catch_unwind(|| {
         oak::set_panic_hook();
         let node = FrontendNode::new();
-        let dispatcher = OakAbiTestServiceDispatcher::new(node);
         let grpc_channel =
             oak::grpc::server::init("[::]:8080").expect("could not create gRPC server pseudo-Node");
-        oak::run_command_loop(dispatcher, grpc_channel.iter());
+        oak::run_command_loop(node, grpc_channel.iter());
     });
 }
 
+oak::impl_dispatcher!(impl FrontendNode : OakAbiTestServiceDispatcher);
```

Comment:
Just curious, why is this syntax chosen?

Reply:
Kind of randomly, I wanted something that reads naturally, but I agree it's not particularly clear right now. Anyways we can (and should) easily change it later on, it should be easy to search & replace, so please feel free to suggest better alternatives.

Reply:
I am actually fine with this syntax. I guess the documentation on the macro is good enough for now.